src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -635,18 +635,22 @@ async function initializeEnvironmentalVariables(frb: FunctionsRuntimeBundle): Pr
     const target = service.replace(/-/g, ".");
     const mode = frb.triggerType === EmulatedTriggerType.BACKGROUND ? "event" : "http";
 
+    let nodeVersion = 0;
+    if (frb.nodeMajorVersion) {
+      // If nodeMajorVersion is set, we ignore pkg.engines.node
```

Comment:
If I'm reading the code correcly, this will still only be set for extensions - maybe mention that in this comment as well?

Reply:
It's also set if the user sets `functions.runtime` in their `package.json` so it's not only for Extensions (although it was before!)